configure.ac

```diff
@@ -73,6 +73,9 @@ AS_IF([test "x$enable_daemon" = "xno"],[SYSTEMD_SYSTEMUNIT=no])
 AC_SUBST(SYSTEMD_SYSTEMUNIT)
 AM_CONDITIONAL(SYSTEMD, test "x${SYSTEMD_SYSTEMUNIT}" != "xno" )
 
+AC_CHECK_FILE([/usr/include/linux/vm_sockets.h],
+	      [], [AC_MSG_ERROR(Cannot find linux/vm_sockets.h)])
+
```

Comment:
It shouldn't stop the build when there is no /usr/include/linux/vm_sockets.h, please just disable the VSOCK in this case.